overlord/devicestate/handlers.go

```diff
@@ -437,8 +466,11 @@ func getSerialRequestConfig(t *state.Task) (*serialRequestConfig, error) {
 		}
 	}
 
-	cfg.requestIDURL = requestIDURL(proxyStore, svcURL)
-	cfg.serialRequestURL = serialRequestURL(proxyStore, svcURL)
+	if proxyURL != nil && svcURL != nil && !newEnoughProxy(proxyURL, client) {
```

Comment:
Can/should we warn here?

Reply:
We can. I'll add logs inside `newEnoughProxy` so we see exactly what went on.

Reply:
is this correct?  did the older proxy versions support the registration  endpoints but not forwarding to a vault that is not the main store?

Reply:
@pedronis yes, exactly